packages/web/src/components/keyboard-shortcuts-dialog.tsx

```diff
@@ -53,6 +53,8 @@ export default function KeyboardShortcutsDialog({
                     description="format code using Prettier"
                   />
                   <ShortcutRow keys={['alt', 'click']} description="go to definition" />
+                  <h5 className="font-semibold pt-6 pb-2">App Builder</h5>
+                  <ShortcutRow keys={['mod', 'shift', 'y']} description="toggle errors panel" />
```

Comment:
There's a keyboard shortcut to open the errors panel - I used the closest relevant one I could find that wasn't already bound from vscode.